beesdoo_pos/models/beesdoo_pos.py

```diff
@@ -4,13 +4,19 @@
 class BeescoopPosPartner(models.Model):
     _inherit = "res.partner"
 
-    def _get_eater(self):
-        eaters = [False, False, False]
+    @api.multi
+    def get_eater(self):
+        self.ensure_one()
+        eaters = [False] * self._get_max_nb_eater_allowed()
         for i, eater in enumerate(self.child_eater_ids):
             eaters[i] = eater.name
```

Comment:
a comment i wrote but forgot to add appparently:

another way to write this:
```python
eaters = [eater.name for eater in self.child_eater_ids)
# pad with False
eaters.extend([False] * (self._get_max_nb_eater_allowed() - len(eaters))
```
not sure which one is more readable, though. 😅